index.js

```diff
@@ -282,7 +282,7 @@ function calculateRange(text, opts, ast) {
 }
 
 function formatRange(text, opts, ast) {
-  if (0 < opts.rangeStart || opts.rangeEnd < text.length) {
+  if (opts.rangeStart > 0 || opts.rangeEnd < text.length) {
```

Comment:
Yeah, I was debating that one too (I was the one who wrote that code originally :stuck_out_tongue:). I'll revert that commit and merge.